test/main.sh

```diff
@@ -616,6 +616,12 @@ run_test test_snap_restore "snapshot restores"
 run_test test_config_profiles "profiles and configuration"
 run_test test_config_edit "container configuration edit"
 run_test test_config_edit_container_snapshot_pool_config "container and snapshot volume configuration edit"
+run_test test_metadata_show "container metadata show"
+run_test test_metadata_edit "contaimer metadata edit"
+run_test test_metadata_template_list "contaimer template list"
+run_test test_metadata_template_content "contaimer template content"
+run_test test_metadata_template_add "container template add"
+run_test test_metadata_template_remove "container template remove"
```

Comment:
Why is it harder? I'd think it's quite the opposite, tests like basic_usage that do a lot of different things are harder to debug when they fail